packages/aws-cdk/package.json

```diff
@@ -86,9 +86,9 @@
     "constructs": "^10.0.0",
     "fast-check": "^3.13.0",
     "jest": "^29.7.0",
+    "jest-mock": "^29.7.0",
```

Comment:
Note, this is already installed with `jest` anyway. Just needs be explicit now, since we are importing a utility function from it.